pkgs/joblib.yaml

```diff
@@ -0,0 +1,7 @@
+# Joblib version 0.9.0b2 for simple prallel python loops.
```

Comment:
Thanks. Is there anything else I have to do here like clicking on "close pull request" or is the rest automatic?